client/ayon_resolve/api/lib.py

```diff
@@ -1226,3 +1229,115 @@ def export_timeline_otio_native(timeline, filepath):
         log.error(f"Failed to export timeline otio: {e}")
         return False
     return True
+
+
+def set_project_fps():
```

Comment:
Bonus points for doing what you had before where it could receive the input argument, in this case the `task_entity` so that we wouldn't need to query it twice whenever we set both project fps and resolution.